supabase/functions/gdk_stats/index.ts

```diff
@@ -0,0 +1,232 @@
+import { createClient } from "https://esm.sh/@supabase/supabase-js@2";
+import { corsHeaders } from "../_shared/cors.ts";
+import { loadEnvVars } from "../_shared/check-env.ts";
+import {
+	GdkStats,
+	Monthly,
+	MonthlyWeather,
+	TreeAdoptions,
+	TreeSpecies,
+	Watering,
+} from "../_shared/common.ts";
+import { GdkError, ErrorTypes } from "../_shared/errors.ts";
+
+const ENV_VARS = ["URL", "SERVICE_ROLE_KEY", "PUMPS_URL"];
```

Comment:
This somehow seems to throw an error for me. I created the .env from env.example but I still get this error

```
supabase functions serve
Setting up Edge Functions runtime...
Serving functions on http://127.0.0.1:54321/functions/v1/<function-name>
Using supabase-edge-runtime-1.54.6 (compatible with Deno v1.43.0)
serving the request with /Users/tomato/Documents/technologiestiftung/giessdenkiez-de-postgres-api/supabase/functions/gdk_stats
runtime has escaped from the event loop unexpectedly: event loop error: Error: Missing environment variables: URL, SERVICE_ROLE_KEY, PUMPS_URL
    at loadEnvVars (file:///Users/tomato/Documents/technologiestiftung/giessdenkiez-de-postgres-api/supabase/functions/_shared/check-env.ts:4:11)
```

Is there a mismatch between the variables in the .env and the ones you are loading?

Reply:
Fixed env confusion in https://github.com/technologiestiftung/giessdenkiez-de-postgres-api/pull/269/commits/d9278356bae958e33b7191ab0b7420640ccbd6e5